faqs/galaxy/Datasets_Data_Privacy_Features.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to set "Data Privacy Features"? 
+area: histories
+box_type: tip
+layout: faq
+contributors: [jennaj,desmond-jasper]
+---
+
+Privacy controls are only enabled if desired. Otherwise, datasets by defaults remain private and unlisted in Galaxy. This means that a dataset you've created is virtually invisible until you publish a link to it.
+
+Below are three optional steps to setting private Histories, a user can make use of any of the options below depending on what the user want to achieve:
+
+1. Changing the privacy settings of individual dataset.
+- Click on the **dataset** name for a dropdown.
+- Clicking the '**pencil - {% icon galaxy-pencil %} icon**
+- Move on the **Permissions** tab.
+- On the permission tab is two input tab
+- On the second input with a label of **access** 
+- Search for the name of the user to grant permission
+- Click on **save permision** 
+- [Watch animation](https://galaxyproject.org/learn/privacy-features/set-perm.gif)
```

Comment:
I am not sure the identation is right here. All the items below 1. should be indented, right?